### ISSUE TITLE
Error: "Unable to UDP Ping" during a test

### ISSUE THREAD

**Issue by richb-hanover** [Zoxc/crusader#53]
Odd error message while collecting data for #52

I was running a succession of GUI client tests. I made the first successful "5GHz - distant" at 7:52:20, then started another and saw this error message.

I restarted the test as usual, and the remainder of tests ran to completion exactly as expected. What other diagnostic information could I provide? Thanks.

```
[2024-08-25 07:52:50] Client version 0.1 running
[2024-08-25 07:52:50] Connected to server 192.168.9.1:35481
[2024-08-25 07:52:51] Idle latency to server 1.21 ms
[2024-08-25 07:52:53] Testing download...
[2024-08-25 07:53:00] Testing upload...
[2024-08-25 07:53:19] Testing both download and upload...
Error: unable to UDP ping

Caused by:
    Can't assign requested address (os error 49)
```

**Comment by richb-hanover** [comment]
Likely fixed in `0.3`

**Comment by Zoxc** [comment]
There's been no change relating to this. It looks like an OS bug though. If it happens often it might be better to treat it like packet loss so it doesn't abort the test.
